gui/default/syncthing/core/syncthingController.js

```diff
@@ -51,13 +51,21 @@ angular.module('syncthing.core')
         $scope.themes = [];
         $scope.globalChangeEvents = {};
         $scope.metricRates = false;
+        $scope.authWarningDismissed = true; 
```

Comment:
Why does this start as true? If will always start off as dismissed, because the localstorage key will be undefined.